src/main/resources/application.yaml

```diff
@@ -38,6 +42,19 @@ spring:
 #          lob:
 #            # silence the 'wall-of-text' - unnecessary exception throw about blob types
 #            non_contextual_creation: true
+  security:
+    oauth2:
+      client:
+        provider:
+          oidc:
+            issuer-uri: ${IDAM_OIDC_URL:http://localhost:5000}/o
```

Comment:
Need to add `IDAM_OIDC_URL` to the chart `values.yaml` too; same for `OIDC_ISSUER` further down.